### PR TITLE
Pud 1174 use recall response instead users endpoint

### PR DESCRIPTION
**Pull request by NataliaOsk**

### CODE REVIEW

File: integration_tests/integration/view-recall.spec.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ context('View a recall', () => {
     })
     recallInformation.assertElementHasText({ qaAttr: 'dossierEmailSentDate', textToFind: '8 September 2021' })
     recallInformation.assertElementHasText({ qaAttr: 'dossierCreatedByUserName', textToFind: 'Bobby Badger' })
+    recallInformation.assertElementHasText({ qaAttr: 'bookedByUserName', textToFind: 'Brenda Badger' })
+    recallInformation.assertElementHasText({ qaAttr: 'assessedByUserName', textToFind: 'Bertie Badger' })
+
     recallInformation.assertElementHasText({ qaAttr: 'recallStatus', textToFind: 'Dossier issued' })
```
Comment (jonnywyatt) [inline integration_tests/integration/view-recall.spec.js:87]: Can we assert for dossierCreatedByUserName somewhere?

File: fake-manage-recalls-api/stubs/__files/get-recall.json [hunk 2]
```diff
@@ -120,5 +120,8 @@
   "dossierCreatedByUserId": "00000000-2222-0000-0000-000000000000",
   "recallAssessmentDueDateTime": "2020-08-06T15:33:57.000Z",
   "dossierTargetDate": "2020-12-14",
-  "status": "RECALL_NOTIFICATION_ISSUED"
+  "status": "RECALL_NOTIFICATION_ISSUED",
```
Comment (jonnywyatt) [inline fake-manage-recalls-api/stubs/__files/get-recall.json:123]: can you re-run `npm run pact-test` to regenerate the contract?

**Review by jonnywyatt — CHANGES_REQUESTED** [review]

File: integration_tests/integration/view-recall.spec.js (hunk 1 above)
Comment (NataliaOsk) [reply integration_tests/integration/view-recall.spec.js:87]: Looks like it was done before on line 83

File: fake-manage-recalls-api/stubs/__files/get-recall.json (hunk 2 above)
Comment (NataliaOsk) [reply fake-manage-recalls-api/stubs/__files/get-recall.json:123]: sure

File: integration_tests/integration/view-recall.spec.js (hunk 1 above)
Comment (jonnywyatt) [reply integration_tests/integration/view-recall.spec.js:87]: sorry I missed that

**Review by jonnywyatt — APPROVED** [review]

**Review by jonnywyatt — APPROVED** [review]

**Pull request merged**